fastly.toml

```diff
@@ -3,3 +3,15 @@ name = "Default starter for Rust"
 description = "Get to know the Fastly Compute@Edge environment with a basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "rust"
+
+[setup]
+  [[setup.backends]]
+    name = "backend_name"
+    prompt = "Origin server able to serve /solutions/ path"
+    address = "developer.fastly.com"
+    port = 443
```

Comment:
I've presumed the Rust code is able to automatically make a request over TLS, but if the user does change the port to `80` for example, then I'm not sure how to tweak the Rust code to support a change in port.